src/Midgard/CreatePHP/Mapper/DoctrineOrmMapper.php

```diff
@@ -37,6 +40,64 @@ class DoctrineOrmMapper extends BaseDoctrineRdfMapper
     /**
      * {@inheritDoc}
      *
+     * In addition to create the object, try to find the parent relation if one exists.
+     */
+    public function prepareObject(TypeInterface $type, $parent = null)
+    {
+        $object = parent::prepareObject($type);
+
+        $config = $type->getConfig();
+        $needParent = isset($config['parent_required']) && (bool) $config['parent_required'];
```

Comment:
i have now created an option for this.

i propose we merge as is - but we lack tests for a lot of this stuff so its hard to say what exactly works or does not...